triedb/pathdb/reader.go

```diff
@@ -29,10 +29,10 @@ import (
 
 // The types of locations where the node is found.
 const (
-	locDirtyCache = "dirty" // dirty cache
-	locCleanCache = "clean" // clean cache
-	locDiskLayer  = "disk"  // persistent state
-	locDiffLayer  = "diff"  // diff layers
```

Comment:
It's OK to use the original order. They are some internal variables and won't be exported to outside at all.